src/modm/platform/clock/sam_pmc/clockgen_impl.hpp.in

```diff
@@ -49,6 +104,19 @@ void ClockGen::enablePllB(uint32_t wait_cycles) {
     // Wait for lock bit
     while(!(PMC->PMC_SR & PMC_SR_LOCKB)) {}
 }
+%% endif
+
+%% if target["family"] in ["v"]
+template<UtmiRefClk refclk>
+void ClockGen::enableUPll(uint32_t wait_cycles) {
+	REG_UTMI_CKTRIM = refclk;
```

Comment:
no template advantage

Reply:
I agree, but I'd leave it as-is for consistency with all other PLL functions. And whoever cleans this up will take care of all of them.